src/optargpm.h

```diff
@@ -525,28 +525,40 @@ OPTARGS *opt_parse (int argc, char **argv, const char *flags, const char *keys,
                     *equals = 0;
                 c = opt_ordinal (o, crepr);
                 if (0==c)
-                    return fprintf (stderr, "Invalid option \"%s\"\n", crepr), (OPTARGS *) 0;
+                {
```

Comment:
This is nit-picking, but since the rest of the code in this file has starting brackets at the same line as the statement I think that should be done here for consistency as well. Also, this is how clang-format would do it with the default llvm setup.

Reply:
done